shopelectro/tests/tests_js_analytics.py

```diff
@@ -87,29 +87,34 @@ class YandexEcommerce(Ecommerce):
 
     fixtures = ['dump.json']
 
-    # @todo #799:120m Test Yandex ecommerce goals.
+    # @todo #808:120m Test Yandex ecommerce goals.
```

Comment:
`# @todo #808:120m Test product add/remove Ya.ecommerce goals.`

Reply:
The issue is created, so I'd prefer edit it